Description: Upstream changes introduced in version 0.1.0-4
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 ctk (0.1.0-4) experimental; urgency=low
 .
   * Changed maintainer and uploader
   * build with  -DCTK_APP_ctkCLIPluginExplorer:BOOL=ON
 .
 The person named in the Author field signed this changelog entry.
Author: Dominique Belhachemi <domibel@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- a/Libs/Visualization/VTK/Core/target_libraries.cmake
+++ b/Libs/Visualization/VTK/Core/target_libraries.cmake
@@ -5,6 +5,5 @@
 #
 
 set(target_libraries
-  VTK_LIBRARIES
   CTKCore
-  )
\ No newline at end of file
+  )
--- a/Libs/Visualization/VTK/Core/CMakeLists.txt
+++ b/Libs/Visualization/VTK/Core/CMakeLists.txt
@@ -3,6 +3,7 @@
 #
 # 3rd party dependencies
 #
+find_package(VTK REQUIRED)
 include(${VTK_USE_FILE})
 
 if(CTK_LIB_Scripting/Python/Core AND CTK_LIB_Scripting/Python/Core_PYTHONQT_USE_VTK)
--- a/Libs/DICOM/Core/ctkDICOMIndexer.cpp
+++ b/Libs/DICOM/Core/ctkDICOMIndexer.cpp
@@ -70,6 +70,14 @@
 
  };
 
+#define DCM_PatientsName DcmTagKey(0x0010, 0x0010)
+#define DCM_PatientsBirthDate DcmTagKey(0x0010, 0x0030)
+#define DCM_PatientsBirthTime DcmTagKey(0x0010, 0x0032)
+#define DCM_PatientsSex DcmTagKey(0x0010, 0x0040)
+#define DCM_PatientsAge DcmTagKey(0x0010, 0x1010)
+#define DCM_PerformingPhysiciansName DcmTagKey(0x0008, 0x1050)
+#define DCM_ReferringPhysiciansName DcmTagKey(0x0008, 0x0090)
+
 
 //------------------------------------------------------------------------------
 static ctkLogger logger("org.commontk.dicom.DICOMIndexer" );
--- a/Libs/DICOM/Core/CMakeLists.txt
+++ b/Libs/DICOM/Core/CMakeLists.txt
@@ -69,6 +69,9 @@
 configure_file( Resources/dcmqrscp.cfg.in dcmqrscp.cfg )
 set (DCMQRSCP_CONFIG ${CMAKE_CURRENT_BINARY_DIR}/dcmqrscp.cfg)
 
+INCLUDE_DIRECTORIES( /usr/include/dcmtk/dcmimgle/ )
+INCLUDE_DIRECTORIES( /usr/include/dcmtk/dcmdata/ )
+
 ctkMacroBuildLib(
   NAME ${PROJECT_NAME}
   EXPORT_DIRECTIVE ${KIT_export_directive}
--- a/Libs/DICOM/Core/ctkDICOMDatabase.cpp
+++ b/Libs/DICOM/Core/ctkDICOMDatabase.cpp
@@ -20,6 +20,16 @@
 
 #include <stdexcept>
 
+
+#define DCM_PatientsName DcmTagKey(0x0010, 0x0010)
+#define DCM_PatientsBirthDate DcmTagKey(0x0010, 0x0030)
+#define DCM_PatientsBirthTime DcmTagKey(0x0010, 0x0032)
+#define DCM_PatientsSex DcmTagKey(0x0010, 0x0040)
+#define DCM_PatientsAge DcmTagKey(0x0010, 0x1010)
+#define DCM_PerformingPhysiciansName DcmTagKey(0x0008, 0x1050)
+#define DCM_ReferringPhysiciansName DcmTagKey(0x0008, 0x0090)
+
+
 // Qt includes
 #include <QDate>
 #include <QDebug>
--- a/Libs/Scripting/Python/Core/CMakeLists.txt
+++ b/Libs/Scripting/Python/Core/CMakeLists.txt
@@ -4,6 +4,8 @@
 # See CTK/CMake/ctkMacroBuildLib.cmake for details
 #
 
+include_directories(/usr/include/PythonQt/)
+
 set(KIT_export_directive "CTK_SCRIPTING_PYTHON_CORE_EXPORT")
 
 # Source files
